gufe/ligandnetwork.py

```diff
@@ -313,9 +348,11 @@ def to_rbfe_alchemical_network(
     #     )
 
     def is_connected(self) -> bool:
-        """Are all ligands in the network (indirectly) connected to each other
+        """Indicates whether all ligands in the network are (indirectly)
```

Comment:
```suggestion
        """Indicates whether all ligands in the network are (directly or indirectly)
```
Maybe unnecessary, but I think this is more accurate.